src/extensions/scratch3_video_sensing/index.js

```diff
@@ -380,11 +254,33 @@ class Scratch3VideoSensingBlocks {
                             defaultValue: 10
                         }
                     }
+                },
+                {
+                    opcode: 'videoToggle',
+                    text: 'turn video [VIDEO_STATE]',
+                    arguments: {
+                        VIDEO_STATE: {
+                            type: ArgumentType.NUMBER,
+                            menu: 'VIDEO_STATE',
+                            defaultValue: 1
+                        }
```

Comment:
I think I'd prefer if this were a string argument (with values like `'on'`) rather than integers. This would make it easier for a human to understand if they read the project JSON, easier to read during debugging, and shouldn't cause a significant performance difference. It also means that if we reorder the items later, existing projects will still work (`'on'` will still mean `'on'` whereas item `1` might become item `2`). Check out the `COLOR_PARAM` menu in the pen extension for an example.

Reply:
:+1: I had just copied the pattern used for the "stage / sprite" menu, should that also be using text instead of numbers?

Reply:
I was just looking at the pen extension to get some ideas how to do this in the menus system, I have to say - It's a bit confusing with how much abstraction is happening in these menu/block definitions... should I be using `format-message` for these things too?  @mzgoddard do you have a better idea how to do this than I?

Reply:
@gnarf I'm not immediately sure. I used numbers following what I thought was the idiom in the music extension.